protos/squareup/cash/pranadb/service/v1/service.proto

```diff
@@ -30,11 +34,34 @@ message Column {
 }
 
 // Issue a query.
-message ExecuteSQLStatementRequest {
+message ExecuteStatementRequest {
```

Comment:
Do ExecuteStatementRequest and ExecutePreparedStatementRequest need to be separate? Wouldn't ExecutePreparedStatementRequest with an empty list in ps_args be equivalent to ExecuteStatementRequest?

Reply:
yeah, I think would be better in one. have updated the pr